src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Navigation/styles.ts

```diff
@@ -35,6 +35,7 @@ export const Navigation = styled.nav<NavigationProps>`
 export const NavigationBarBrand = styled(Link)`
   /* padding: 1rem; */
```

Comment:
While you are at it, can you remove the comment as well?

Reply:
Sure thing! :+1: